integration/src/test/scala/com/snowplowanalytics/snowplow/collectors/scalastream/integration/utils/Base64.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2013-2022 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0, and
+ * you may not use this file except in compliance with the Apache License
+ * Version 2.0.  You may obtain a copy of the Apache License Version 2.0 at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the Apache License Version 2.0 is distributed on an "AS
+ * IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the Apache License Version 2.0 for the specific language
+ * governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.collectors.scalastream.integration.utils
+
+import org.apache.commons.codec.binary.{Base64 => ABase64}
```

Comment:
Why not using directly `import java.util.Base64` ?

Reply:
Can't remember the reason, but it wasn't working as expected.